test/otherTools.cpp

```diff
@@ -252,11 +253,12 @@ TEST(networkTools, getNetworkInterfaces)
   }
 }
 
-TEST(networkTools, getBestPublicIp)
-{
-  using kiwix::getBestPublicIp;
+std::ostream& operator<<(std::ostream& os, const kiwix::IpAddress& ip) {
+    return os << "[" << ip.addr << ", " << ip.addr6 << "]";
+}
```

Comment:
Questioning whether we should keep this.

Reply:
That's a minor issue. Proceed however you like.